src/main/java/org/mockito/internal/junit/DefaultStubbingLookupListener.java

```diff
@@ -20,6 +20,7 @@
 class DefaultStubbingLookupListener implements StubbingLookupListener {
 
     Strictness currentStrictness;
+    boolean mismatchesReported;
```

Comment:
I'm an anti setter getter person. But on the other hand I don't like leaking mutable variables.